wp-content/civi-extensions/goonjcustom/Civi/GoonjInitiatedEventsService.php

```
@@ -38,6 +42,199 @@ public static function getSubscribedEvents() {
     ];
   }
 
+  public static function handleEventMonetaryContributionDelete(string $op, string $objectName, $objectId, &$params) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$params'. (Unused Code Rules)

(UnusedFormalParameter)